src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
```

Comment:
Maybe we should accept info about users as well as about their bets in this class?

Reply:
added